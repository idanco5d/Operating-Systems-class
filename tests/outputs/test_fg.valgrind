==6457== Memcheck, a memory error detector
==6457== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==6457== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==6457== Command: /mnt/c/Users/illus/os/hw/1/smash
==6457== Parent PID: 6452
==6457== 
==6457== 
==6457== FILE DESCRIPTORS: 4 open (3 std) at exit.
==6457== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_fg.valgrind
==6457==    <inherited from parent>
==6457== 
==6457== 
==6457== HEAP SUMMARY:
==6457==     in use at exit: 0 bytes in 0 blocks
==6457==   total heap usage: 45 allocs, 45 frees, 80,656 bytes allocated
==6457== 
==6457== All heap blocks were freed -- no leaks are possible
==6457== 
==6457== For lists of detected and suppressed errors, rerun with: -s
==6457== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
