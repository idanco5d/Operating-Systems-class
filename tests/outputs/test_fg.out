smash> smash> smash> sleep 1& : 9180
smash> smash> 